packages/components/src/Banner/Banner.mdx

```diff
@@ -110,10 +110,12 @@ possibly unknown impact a user’s changes or actions will have. They should not
 begin with "Warning:" and should not be used as a way to block or redirect a
 workflow.
 
+You should use [ConfirmationModal](/components/confirmation-modal) when you need to
+get explicit confirmation from the user before they complete an action.
```

Comment:
for actions that are difficult to reverse? perhaps?

Reply:
ah good

Reply:
```suggestion
get explicit confirmation from the user before they complete an action that is difficult to reverse.
```